code/modules/reagents/reagent_containers/blotter.dm

```diff
@@ -0,0 +1,5 @@
+/obj/item/reagent_containers/pill/blotter
+	name = "Blotter"
+	desc = "Paper soaked in a reagent you can take via mouth."
+	icon_state = "blotter"
+	volume = 50
```

Comment:
Big volume for just a little piece of paper

Reply:
lots of drugs to take, plus you cant remove the reagent from the blotter without diluting it